tests/.deprc

```diff
@@ -0,0 +1,4 @@
+  {
+    "root": "tests",
```

Comment:
Assuming you could also just do `./` if you're opening this up in its own directory